dbt_af/conf/config.py

```diff
@@ -5,6 +5,30 @@
 import pendulum
 
 
+@attrs.define(frozen=True)
+class AfCallbacksConfig:
```

Comment:
Let's rename it to `CustomAfCallbacksConfig`

Reply:
class was renamed to `CustomAfCallbacksConfig`